pkgs/os-specific/darwin/apple-sdk/frameworks.nix

```diff
@@ -2,79 +2,79 @@
   AVFoundation            = [ ApplicationServices CoreGraphics ];
   AVKit                   = [];
   Accounts                = [];
-  AddressBook             = [ Carbon CF ];
+  AddressBook             = [ Carbon ];
   AppKit                  = [ AudioToolbox AudioUnit Foundation QuartzCore ];
   AppKitScripting         = [];
   AppleScriptKit          = [];
   AppleScriptObjC         = [];
-  AudioToolbox            = [ CoreAudio CF CoreMIDI ];
-  AudioUnit               = [ AudioToolbox Carbon CoreAudio CF ];
+  AudioToolbox            = [ CoreAudio CoreMIDI ];
+  AudioUnit               = [ AudioToolbox Carbon CoreAudio ];
   AudioVideoBridging      = [ Foundation ];
   Automator               = [];
-  CFNetwork               = [ CF ];
+  CFNetwork               = [];
   CalendarStore           = [];
   Cocoa                   = [ AppKit ];
   Collaboration           = [];
   # Impure version of CoreFoundation, this should not be used unless another
   # framework includes headers that are not available in the pure version.
   CoreFoundation          = [];
-  CoreAudio               = [ CF IOKit ];
+  CoreAudio               = [ IOKit ];
   CoreAudioKit            = [ AudioUnit ];
   CoreData                = [];
-  CoreGraphics            = [ Accelerate CF IOKit IOSurface SystemConfiguration ];
-  CoreImage               = [ ];
+  CoreGraphics            = [ Accelerate IOKit IOSurface SystemConfiguration ];
+  CoreImage               = [];
   CoreLocation            = [];
-  CoreMIDI                = [ CF ];
+  CoreMIDI                = [];
   CoreMIDIServer          = [];
-  CoreMedia               = [ ApplicationServices AudioToolbox AudioUnit CoreAudio CF CoreGraphics CoreVideo ];
-  CoreMediaIO             = [ CF CoreMedia ];
-  CoreText                = [ CF CoreGraphics ];
-  CoreVideo               = [ ApplicationServices CF CoreGraphics IOSurface OpenGL ];
+  CoreMedia               = [ ApplicationServices AudioToolbox AudioUnit CoreAudio CoreGraphics CoreVideo ];
+  CoreMediaIO             = [ CoreMedia ];
+  CoreText                = [ CoreGraphics ];
+  CoreVideo               = [ ApplicationServices CoreGraphics IOSurface OpenGL ];
   CoreWLAN                = [ SecurityFoundation ];
   DVDPlayback             = [];
-  DirectoryService        = [ CF ];
-  DiscRecording           = [ CF CoreServices IOKit ];
+  DirectoryService        = [];
+  DiscRecording           = [ CoreServices IOKit ];
   DiscRecordingUI         = [];
-  DiskArbitration         = [ CF IOKit ];
+  DiskArbitration         = [ IOKit ];
   EventKit                = [];
   ExceptionHandling       = [];
   FWAUserLib              = [];
-  ForceFeedback           = [ CF IOKit ];
-  Foundation              = [ cf-private libobjc Security ApplicationServices SystemConfiguration ];
-  GLKit                   = [ CF ];
+  ForceFeedback           = [ IOKit ];
+  Foundation              = [ libobjc CoreFoundation Security ApplicationServices SystemConfiguration ];
```

Comment:
I think that with this we can actually get rid of most of the remaining cf-private references, as well as some of the [ugly cflags](https://search.nix.gsc.io/?q=%5C%24%7BCoreFoundation%7D%2F&i=nope&files=&repos=) workaround needed before because of missing symbols.